arrows/ffmpeg/tests/test_open_video.cxx

```diff
@@ -0,0 +1,167 @@
+/*ckwg +29
```

Comment:
Maybe rename file to `test_video_input_ffmpeg.cxx` since the tests cover more than just opening the video.